packages/bff/src/entities/Profile.ts

```diff
@@ -9,11 +18,17 @@ export class Profile {
   @Column({ length: 255, nullable: true })
   language: string;
 
+  @OneToMany('SavedSearch', 'profile')
+  savedSearches: SavedSearch[];
```

Comment:
Do we need a OneToMany here? Or is it enough to just have `sub` as its own field